files/build_templates/host_regex.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "tag": "disk-usage",
+        "regex": ".([a-zA-Z0-9-_]*). space usage (\d+\.\d+)% matches resource limit .space usage.(\d+\.\d+)%.",
+        "params": [ "fs", "usage", "limit" ]
+    },
+    {
+        "tag": "memory-usage",
+        "regex": "(?:mem|cpu user) usage of (\d+\.\d+)% matches resource limit .(?:mem|cpu user) usage>(\d+\.\d+)%.",
```

Comment:
Report only mem usage. 
Need a different tag for cpu usage
If we don't have an event, you can create one